packages/docs/render-blocks.md

```diff
@@ -0,0 +1,109 @@
+# Render Blocks
+
+In block suite, blocks can be rendered by any UI framework.
```

Comment:
Please kindly stick to using the term "BlockSuite" in docs 😄

Reply:
Got it